geps/gep-713.md

```diff
@@ -1,29 +1,49 @@
-# GEP-713: Policy Attachment
+# GEP-713: Metaresources and Policy Attachment
 
 * Issue: [#713](https://github.com/kubernetes-sigs/gateway-api/issues/713)
 * Status: Experimental
 
 ## TLDR
 
-This GEP aims to standardize policy attachment to resources associated with
-Gateway API by establishing a pattern which defines how `Policy` API types can
-have their relevant effects applied to network traffic. Individual policy APIs
-(e.g. `TimeoutPolicy`, `RetryPolicy`, etc) will include a common `TargetRef`
-field in their specification to identify how and where to apply that policy.
-This will be important for providing a consistent experience across
-implementations of the API, even for configuration details that may not be fully
-portable.
+This GEP aims to standardize terminology and processes around using one Kubernetes
+object modify the functions of one or more other objects.
+
+This GEP defines some terms, firstly: _Metaresource_.
+
+A Kubernetes object that that _augments_ the behavior of an object
+in a standard way is called a _Metaresource_ are a type of object. ReferenceGrant
+is an example of this general type of metaresource, but it is far from the only
+one.
+
+This document proposes controlling the creation of configuration in the underlying
+Gateway data plane using two types of Policy Attachment.
+A "Policy Attachment" is a specific type of _metaresource_ that can affect specific
+settings across either one object (this is "Direct Policy Attachment"), or objects
+in a hierarchy (this is "Hierarchical Policy Attachment").
+
+Individual policy APIs:
+- must be their own CRDs (e.g. `TimeoutPolicy`, `RetryPolicy` etc),
+- can be included in the Gateway API API group and installation or be defined by
+  implementations
+- and must include a common `TargetRef` struct in their specification to identify how and where to
+  apply that policy, along with a field that indicates if the reference is `Direct`
```

Comment:
I think we discussed removing the "field indicating the reference type" part?

Reply:
Thanks, removed.